common/reviews/api/tools.api.md

```diff
@@ -1978,6 +2013,54 @@ interface ICamera {
     viewUp?: Point3;
 }
 
+// @public (undocumented)
+interface IContour {
+    // (undocumented)
+    color: any;
+    // (undocumented)
+    getColor(): Point3;
+    // (undocumented)
+    getFlatPointsArray(): number[];
+    getPoints(): Point3[];
+    // (undocumented)
+    _getSizeInBytes(): number;
+    // (undocumented)
+    getType(): ContourType;
+    // (undocumented)
+    readonly id: string;
+    // (undocumented)
+    points: Point3[];
+    // (undocumented)
+    readonly sizeInBytes: number;
+    // (undocumented)
+    type: ContourType;
+}
+
+// @public (undocumented)
+interface IContourSet {

```

Comment:
Should this have frame of reference UID to indicate what space the contour data is in?